src/app.vue

```diff
@@ -3,24 +3,98 @@
     <nav>
       <div class="inner">
         <router-link class='logo' to='/'>
-          <img src="@/assets/header-logo.svg" alt=""/>
-          <span>explorer</span>
+          <img src="@/assets/header-logo.svg" alt="" />
+          <span class="app-name">explorer</span>
         </router-link>
         <div class="links">
           <router-link class='link' to='/'>
             Dashboard
           </router-link>
-          <router-link class='link'  to='/blocks'>
+          <router-link class='link' to='/blocks'>
             Blocks
           </router-link>
           <router-link class='link' to='/tx'>
             Transactions
           </router-link>
         </div>
+        <div class="mobile-nav">
+          <div class="pageName">
+            <strong v-if="$route.name === 'Index'">Dashboard</strong>
+            <strong v-if="$route.name === 'BlockList'">Blocks</strong>
+            <strong v-if="$route.name === 'Transaction'">Transactions</strong>
+          </div>
+          <div class="burger" @click='expandMenu'>
+            <img src="@/assets/burger.svg" alt="" />
+          </div>
+          <div class="mobile-nav-fixed" :class="{ active: isOpened}" @click="expandMenu">
+            <div class="mobile-nav-fixed-holder">
+              <div class="inner">
+                <div class="mobile-nav-fixed-action" @click='this.isOpened = false' >
+                  <span>
+                    <img src="@/assets/close.svg" alt=""/>
+                  </span>
+                </div>
+                <div class="mobile-nav-fixed-main-links">
+                  <router-link to='/'>
+                    Dashboard
+                  </router-link>
+                  <router-link to='/blocks'>
+                    Blocks
+                  </router-link>
+                  <router-link to='/tx'>
+                    Transactions
+                  </router-link>
+                </div>
+                <div class="mobile-nav-fixed-secondary-links">
+                  <router-link to='/'>
+                    imprint
+                  </router-link>
+                  <router-link to='/'>
+                    get started
+                  </router-link>
+                  <router-link to='/'>
+                    aeternity.com
+                  </router-link>
+                </div>
+                <div class="mobile-nav-fixed-socials-links">
+                  <div class="mobile-nav-fixed-socials-links-row">
```

Comment:
Is there any reason to use the class name of the parent node as a prefix for the class name of child node? Can we use unprefixed class names with selector nesting in SCSS instead of this?
```scss
.mobile-nav {
  .fixed {
    .socials-links {
      .row {
        // ...
      }
    }
  }
}
```

Reply:
no reason at all, i just follow BEM like style but ofc i can try to use other class names.
Only reason that i use BEM is because when i write style declaration its more fast if we use only one selector. etc:
.parent .child {} is slower than .parent_child {} because browser read from right to left declarations.
but its not problem for me to stick to the plan